### PR TITLE
Removed silly assignments, project-wide

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Removed silly assignments, project-wide

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

File: libraries/legacy/request/request.php [hunk 1]
```diff
@@ -528,7 +528,6 @@ protected static function _cleanVar($var, $mask = 0, $type = null)
 		if ($mask & 2)
 		{
 			// If the allow raw flag is set, do not modify the variable
-			$var = $var;
```
Comment (C-Lodder) [inline libraries/legacy/request/request.php]: If you're going to remove this line, you may aswell remove the entire `if` statement then replace the next `elseif` with `if`

**Review by billouboq — COMMENTED** [review]
in libraries/legacy/request/request.php

I think we can get rid of the if ($mask & 2)

**Comment by frankmayer** [comment]
@billouboq Actually not, because then $var will be changed in the `else` part.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on f8d514d49944abfd2039f291c436d7d653938467<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13196">issues.joomla.org/tracker/joomla-cms/13196</a>.</sub>

**Comment by shur** [comment]
I have tested this item :white_check_mark: successfully on f8d514d49944abfd2039f291c436d7d653938467<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13196">issues.joomla.org/tracker/joomla-cms/13196</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13196">issues.joomla.org/tracker/joomla-cms/13196</a>.</sub>

**Pull request merged**